Ghidra/Features/Decompiler/src/decompile/cpp/database.cc

```diff
@@ -2879,9 +2879,10 @@ void Database::clearResolve(Scope *scope)
     res = resolvemap.find(rng.getFirstAddr());
     while(res.first != res.second) {
       if ((*res.first).scope == scope) {
-	resolvemap.erase(res.first);
```

Comment:
p.s. it was originally indented using Tab. Updated to Spaces so style looks better.